Code quality test failing at master branch.

Currently on master `bin/test quality` is having an error: 
```py
ahappyidiot@Ahappyidiot:~/Desktop/Gsoc/sympy$ bin/test quality
================================================================= test process starts =================================================================
executable:         /home/ahappyidiot/anaconda2/bin/python  (2.7.11-final-0) [CPython]
architecture:       64-bit
cache:              yes
ground types:       python 
random seed:        50269568
hash randomization: on (PYTHONHASHSEED=2030885458)

sympy/utilities/tests/test_code_quality.py[6] F.....                                                                                             [FAIL]

_______________________________________________________________________________________________________________________________________________________
________________________________________________ sympy/utilities/tests/test_code_quality.py:test_files ________________________________________________
  File "/home/ahappyidiot/Desktop/Gsoc/sympy/sympy/utilities/tests/test_code_quality.py", line 227, in test_files
    check_directory_tree(BIN_PATH, test, set(["~", ".pyc", ".sh"]), "*")
  File "/home/ahappyidiot/Desktop/Gsoc/sympy/sympy/utilities/tests/test_code_quality.py", line 107, in check_directory_tree
    check_files(glob(join(root, pattern)), file_check, exclusions)
  File "/home/ahappyidiot/Desktop/Gsoc/sympy/sympy/utilities/tests/test_code_quality.py", line 123, in check_files
    file_check(fname)
  File "/home/ahappyidiot/Desktop/Gsoc/sympy/sympy/utilities/tests/test_code_quality.py", line 146, in test
    test_this_file(fname, test_file)
  File "/home/ahappyidiot/Desktop/Gsoc/sympy/sympy/utilities/tests/test_code_quality.py", line 162, in test_this_file
    assert False, message_test_suite_def % (fname, idx + 1)
AssertionError: Function should start with 'test_' or '_': /home/ahappyidiot/Desktop/Gsoc/sympy/bin/generate_test_list.py, line 57

================================================= tests finished: 5 passed, 1 failed, in 0.07 seconds =================================================
DO *NOT* COMMIT!
```

--- a/sympy/utilities/tests/test_code_quality.py
+++ b/sympy/utilities/tests/test_code_quality.py
@@ -44,7 +44,7 @@ gen_raise_re = re.compile(
 old_raise_re = re.compile(r'^\s*(>>> )?(\.\.\. )?raise((\s*\(\s*)|\s+)\w+\s*,')
 test_suite_def_re = re.compile(r'^def\s+(?!(_|test))[^(]*\(\s*\)\s*:$')
 test_ok_def_re = re.compile(r'^def\s+test_.*:$')
-test_file_re = re.compile(r'.*test_.*\.py$')
+test_file_re = re.compile(r'.*[/\\]test_.*\.py$')
 
 
 def tab_in_leading(s):

